src/status_im2/contexts/add_new_contact/views.cljs

```diff
@@ -96,8 +95,9 @@
          {:type       :outline
           :icon-only? true
           :size       40
-          :on-press   #(rf/dispatch [::qr-scanner/scan-code
-                                     {:handler :contacts/qr-code-scanned}])}
+          :on-press   #(rf/dispatch [:open-modal :scan-address
+                                     {:show-subtitle?  false
+                                      :bottom-padding? false}])}
```

Comment:
its little weird, we have to pass this param. Are there any instances of scanner, where we are showing subtitle and different bottom padding. Can't we just hard code them?

Reply:
@Parveshdhull creating a new space to scan profile qr code addresses concerns you highlighted in this comment